README.md

```diff
@@ -1,2 +1,62 @@
-# COP27
-This repo is a webpage for The World Climate Summit COP27 event.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Identity
+
+## FIRST MICROVERSE CAPSTONE PROJECT, WORLD CLIMATE SUMMIT
+>  COP27
+This repo is a webpage for Climate Action Innovation Zone COP27 event.
+
+## Built With
+- HTML
+- CSS
+- GIT AND GITHUB
+- JAVASCRIPT (DOM AND EVENT LISTENERS)
+
+
+## Getting Started
+
+**Instructions on how to view and clone my project locally.**
+ To Access my project locally please follow the following instructions;
+1. Create a folder on your computer where you will clone the project
+2. Enter this url: https://github.com/dialite/COP27/tree/Climate-Summit in your web browser.
+3. Once opened, move your cursor to the top left of the project page to select 'code' a green code button will be visible on the righthand corner showing download    options.
+5. Select download Zip option from drop down menu to your desired folder on your computer.
+6. Once the download is complete you will be able to access my project locally.
+7. Also, you can navigate to your folder using a console.
+8. Then run git clone https://github.com/dialite/COP27/tree/Climate-Summit
+
+Here is a link directly to the repository page: https://github.com/dialite/COP27/tree/Climate-Summit
+
+
+## Live Demo (if available)
+
+(https://www.loom.com/share/3cc3451156eb4f04978249a01629c086)
+
+Here is a link to the live page: https://dialite.github.io/COP27/
+
+
+👤 Raymond Olisa
+
+- GitHub: [@dialite](https://github.com/dialite)
+- Twitter: [@raymondolisa01](https://twitter.com/raymondolisa01)
+- LinkedIn: [raymond-olisa](https://www.linkedin.com/in/raymond-olisa-775929243/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Cindy Shin called CC Global Summit 2015 - Design Initiator
+- Font Awesome
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you give the appropriate credit to Cindy and why not provide the `link` to his/her Behance account :smile: